google/cloud/forseti/services/inventory/base/resources.py

```
@@ -1821,12 +1905,29 @@ def iter(self):
                 except ResourceNotSupported as e:
                     # API client doesn't support this resource, ignore.
                     LOGGER.debug(e)
+                except Exception as e:  # pylint: disable=broad-except
```

why is this needed?

---

Broad except is needed to catch all exceptions other than ResourceNotSupported while iterating through the org policies.

---

This was not here before, we do have other exception handling on a higher level.

---

I later realized this and had uncommented the broad exception. Removed now.